console/src/components/Services/About/About.js

```diff
@@ -30,13 +30,32 @@ class About extends Component {
           latestServerVersion: latest.latest,
         })
       );
+    fetch('http://localhost:8080/v1/query', {
```

Comment:
- Add an empty line above this for readability
- The endpoint should be the variable `Endpoints.query` instead of `localhost...`